runtime-common/realm.ts

```diff
@@ -710,6 +718,18 @@ export class Realm {
     });
   }
 
+  private async getRealmInfo(): Promise<Response> {
+    return new Response(
+      JSON.stringify({
+        data: {
+          id: this.url,
+          type: "realm-info",
+          attributes: { baseRealm: this.baseRealmURL, url: this.url },
+        },
+      })
+    );
+  }
```

Comment:
we've added a new endpoint that can return info about a realm by tacking `/_realmInfo` to the end of any module's URL. this will be able to describe the realm URL that governs the module as well as the base realm URL that the realm is using.